src/util.iced

```diff
@@ -0,0 +1,14 @@
+
+#---------------------------------------------------------
+# Constant-time buffer comparison
+#
+exports.bufeq_secure = (x,y) ->
+  ret = if not x? and not y? then true
+  else if not x? or not y? then false
+  else if x.length isnt y.length then false
+  else
+    check = 0
+    for i in [0...x.length]
+      check |= (x.readUInt8(i) ^ y.readUInt8(i))
+    (check is 0)
+  return ret
```

Comment:
well i wrote this a while ago.  I'd honestly be more worried about using a node version with a polyfill on browserify that might have a bug in it.  I feel perfectly comfortable with this as is.  Unless you see a bug which I don't.